server/container/guice/configuration/src/main/java/org/apache/james/utils/PropertiesProvider.java

```diff
@@ -85,7 +88,9 @@ private Configuration getConfiguration(File propertiesFile) throws Configuration
 
     private Optional<File> getConfigurationFile(String fileName) {
         try {
-            return Optional.of(fileSystem.getFile(configurationPrefix.asString() + fileName + ".properties"))
+            File file = fileSystem.getFile(configurationPrefix.asString() + fileName + ".properties");
+            LOGGER.info("Load configuration file {}", file.getAbsolutePath());
```

Comment:
Maybe it deserves to be a debug?

Reply:
Configurations are loaded once ?!? during first use so it's not a performance penalty.
It is valuable to see them loading and from where. 
We can always change if you feel strongly about it. 

My biggest pain related to this is that if the path to the logging config is wrong, we don't get any logging at all. Actually we do get some logging - which leads me to think some logging bridge is not setup ok.

Reply:
I'm resolving this with logging kept at INFO.
Please reopen if you feel strongly it should be otherwise.